ra/ra_test.go

```diff
@@ -575,15 +580,29 @@ func TestUpdateRegistrationSame(t *testing.T) {
 func TestNewAuthorization(t *testing.T) {
 	_, sa, ra, _, cleanUp := initAuthorities(t)
 	defer cleanUp()
-	_, err := ra.NewAuthorization(ctx, AuthzRequest, 0)
-	test.AssertError(t, err, "Authorization cannot have registrationID == 0")
+
+	// This tests that there is a foreign key constraint on the authorization table
```

Comment:
Rather than commenting this, we should delete it. Also, can you check whether the SA already has an equivalent test case, and if not, add one in a separate PR?